frontend/lib/location/determine_position.dart

```diff
@@ -19,36 +23,55 @@ Future<Position> determinePosition() async {
 /// When the location services are not enabled or permissions
 /// are denied the `Future` will return with an `PositionNotAvailableException`.
 /// Else it will complete without a value.
-Future<void> requestPermissionToDeterminePosition() async {
+Future<void> requestPermissionToDeterminePosition({bool userInteract}) async {
   var serviceEnabled = await Geolocator.isLocationServiceEnabled();
   if (!serviceEnabled) {
-    throw PositionNotAvailableException('Location services are disabled.');
+    if (userInteract) {
+      await Geolocator.openLocationSettings();
```

Comment:
This feels a bit weird to me if I imagine I am an average user.

I denied the permission and close the app. One week later I open it and click on the locaiton button or the "Search nearby button". Then the location settings are opened which is this screen on android for me:

![image](https://user-images.githubusercontent.com/905221/107345750-32011c80-6ac4-11eb-9c35-1afcc6d7db61.png)


Now I do not know what to do or what the app wants from me. Instructions should be given on how to enable permissions in my opinion.

Reply:
I added instructions.